api/app/signals/apps/signals/managers.py

```diff
@@ -147,7 +147,7 @@ def split(self, split_data, signal, user=None):  # noqa: C901
 
                 reporter_data = {'_signal': child_signal}
                 reporter_data.update({
-                    k: getattr(parent_signal.reporter, k) for k in ['email', 'phone', 'is_anonymized']  # noqa
+                    k: getattr(parent_signal.reporter, k) for k in ['email', 'phone', 'email_anonymized', 'phone_anonymized']  # noqa
```

Comment:
Split after a long period will retain user data longer ... we may need to take that up with the business.